code/modules/mob/living/carbon/carbon.dm

```diff
@@ -591,17 +591,19 @@ GLOBAL_LIST_INIT(ventcrawl_machinery, list(/obj/machinery/atmospherics/unary/ven
 				hurt = FALSE*/
 	if(hit_atom.density && isturf(hit_atom))
 		if(hurt)
-			Weaken(2 SECONDS)
-			take_organ_damage(10)
+			visible_message("<span class='danger'>[src] slams into [hit_atom]!</span>", "<span class='userdanger'>You slam into [hit_atom]!</span>")
+			take_organ_damage(7.5 * speed) // speed while thrower is standing still is 2, while walking with an aggressive grab is 2.4
+			KnockDown(2 SECONDS)
```

Comment:
I would also considering increasing the knockdown values slightly I maybe something like 3 seconds?